src/Symfony/Component/Messenger/Command/DebugCommand.php

```diff
@@ -78,24 +80,52 @@ protected function execute(InputInterface $input, OutputInterface $output)
         foreach ($mapping as $bus => $handlersByMessage) {
             $io->section($bus);
 
-            $tableRows = [];
-            foreach ($handlersByMessage as $message => $handlers) {
-                $tableRows[] = [sprintf('<fg=cyan>%s</fg=cyan>', $message)];
-                foreach ($handlers as $handler) {
-                    $tableRows[] = [
-                        sprintf('    handled by <info>%s</>', $handler[0]).$this->formatConditions($handler[1]),
-                    ];
-                }
+            $this->describeMessages($handlersByMessage, $bus, $io);
+            $this->describeMiddlewares([], $bus, $io);
+        }
+    }
+
+    private function describeMessages($handlersByMessage, string $bus, SymfonyStyle $io): void
```

Comment:
You can also declare `$io` private in the class and instantiate `SymfonyStyle` in the initialize method to avoid having a pass as a parameter.
WDYT ?